demo/grpc-api/src/main/app.d/crypto-application.app

```diff
@@ -4,3 +4,5 @@ id=app.deephaven.demo.crypto.Application
 name=Crypto Application
 enabled=true
 file_0=crypto-application.py
+file_1=tensorflow-install.py
```

Comment:
Should we add the install to the Dockerfile instead? As per https://github.com/deephaven/deephaven-core/blob/demo/demo/grpc-api/src/main/app.d/crypto-application.py
My reasoning:
- Not related to the crypto app (I don't think), so shouldn't be referenced from crypto-application.app. Would need to make a separate app just for this
- Package should be installed with the image, so that it doesn't need to be installed whenever the session starts up (which could slow down startup)

@devinrsmith may have input as well

Reply:
agreed. I was going for the quick and dirty. removed that for the better option which is more extendable. now have a requirements file to pip install in the Dockerfile.